src/Watchers/ExceptionWatcher.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace Taecontrol\Larvis\Watchers;
+
+use Exception;
+use Throwable;
+use Taecontrol\Larvis\Larvis;
+use Illuminate\Support\Facades\Event;
+use Illuminate\Log\Events\MessageLogged;
+use Taecontrol\Larvis\ValueObjects\Data\ExceptionData;
+
+class ExceptionWatcher extends Watcher
+{
+    public function register(): void
+    {
+        $this->enabled = config('larvis.watchers.exceptions.enabled');
+
+        Event::listen(function (MessageLogged $message) {
+            if (! $this->enabled()) {
+                return;
+            }
+
+            if (! $this->isAnException($message)) {
+                return;
+            }
+
+            $this->handleException($message->context['exception']);
+        });
+    }
+
+    public function isAnException(MessageLogged $message): bool
```

Comment:
do we need this method as public?

Reply:
I am going to change this method to private, currently there is no need or use for this function as public.